plugins/woocommerce/client/blocks/assets/js/blocks/mini-cart/iapi-frontend.ts

```diff
@@ -636,9 +636,16 @@ const { state: cartItemState } = store(
 			},
 
 			*changeQuantity(): Generator< unknown, void > {
+				const variation = cartItemState.cartItem.variation.map(
+					( { raw_attribute, ...rest } ) => ( {
+						...rest,
+						attribute: raw_attribute,
+					} )
+				);
```

Comment:
We need to override the `attribute` property because the response from the API has a different format than the object used in `addCartItem`.

The `variation` of the REST API has this format:

```json
{
    "raw_attribute": "attribute_pa_color",
	"attribute": "Color",
	"value": "Red"
}
```

While the `variation` in the `addCartItem` has this one:

```json
{
	"attribute": "attribute_pa_color",
	"value": "Red"
}
```

 I briefly discussed it with @Aljullu, it could make sense to unify both and change `attribute` to `selected_attribute`. Additionally, the descriptive attribute name ("Color"), is not used right now, but it could be used in other blocks like Cart and Checkout.

Anyways, it can be handled in another PR.

Reply:
We should create an issue and remember to do it before marking the stores as public so that a change like this is not a breaking change.